data/coalition/coalition missions.txt

```diff
@@ -143,3 +143,315 @@ mission "Coalition License Patch"
+
+
+
+mission "Coalition Yottrite 5"
+	name "Saryd Specialists"
+	description "Transport the twins Nasilor and Alituri to <destination>, where they will assist in the Yottrite research."
+	passengers 2
+	to offer
+		has "Coalition Yottrite 4: done"
+	source "Shadowed Valley"
+	destination "Delve of Bloptab"
+	on offer
+		conversation
+			`A Saryd wearing their version of a lab coat is waiting for you when you enter the spaceport.`
+			`	"Working with Lugglop, you are?" He asks, and you nod. "Nasilor, my name is. Packing up our belongings, my sister is. Bring you to our laboratory, I shall."`
+			`	You follow him deep into the underground spaceport to the "residential" section, which, to you, is indistinguishable from a research compound. The endless doors to your left and right occasionally give space for massive glass panels, giving you a nice view of the massive crystal formations here.`
+			`	Nasilor stops by a door immediately after one of these "viewing sites" and opens it. He introduces you to his sister and you two start helping her pack what remains. Most of what they're bringing are several research papers, though you can't figure out what's written on them.`
+			`	Finally, you start heading back to your ship. Noticing your interest in the beauty of the gypsum caves, the two Saryd researchers slow down their pace and start to tell you more about themselves. They are twins, both born on Saros, but have been working here for decades. Though you don't understand much of the technical jargon, you figure out that Nasilor specializes in cryptocrystalline compounds, while Alituri focuses her studies on planetary and cosmic conditions that lead to the formation of the gypsum caves here.`
+			`	With the little "tour" over, when you three reach your ship, you show them to their bunks and prepare to head back to <planet>.`
+				accept
+
+
+
+mission "Coalition Yottrite 6"
+	name "Master Shipwright"
+	description "Head to <destination> to pick up a member of House Mallob so that they may help in trying to apply Yottrite to an alloy."
+	landing
+	to offer
+		has "Coalition Yottrite 5: done"
+	source "Delve of Bloptab"
+	destination "Market of Gupta"
+	on offer
+		conversation
+			`Lugglop is waiting for you when you land and greets the two researchers as you help them get their things out of the ship.`
+			`	He points them to a hotel where he had a room prepared for them, and asks you to follow him to his office again.`
+			`	"Filled to the brim with cutting edge equipment, our facilities here and on the neighbor planet are. Alas, venture much into the theoretical field needed to study such rare elements, our personnel here does not."`
+			`	He shows you to his map again, this time pointing to a nearby Arachi world. "Once cracked down some aspects of Yottrite, the twins have, an expert in ship making, we will need," he says. "On <destination>, prepared one of their representatives, House Mallob has. Bring them here, you must."`
```

Comment:
```suggestion
			`	He shows you to his map again, this time pointing to a nearby Arach world. "Once cracked down some aspects of Yottrite, the twins have, an expert in ship making, we will need," he says. "On <destination>, prepared one of their representatives, House Mallob has. Bring them here, you must."`
```

Reply:
the map label spells out `Arachi`, so when referring to one of their worlds being shown in a map I'd say using `Arachi` is better.

on a similar not Wizrad questioned in PMs whether Kimek or Kimeks was plural, I was using the latter but the examples in the game currently use the former, and given the label does too I'll be changing mine to fit

Reply:
Still, it's "human world", "wanderer world" etc, not "humans world". Becaus of that, I'm still convinced it should be Arach world

Reply:
I'd argue that's because both of those follow the conventional "plural = word with s at the end" for when referring to their species as a group, while with the Arachi it's different, so not feeling it much for this change

Reply:
It doesn't matter how the plural of a word is, the rule is always the same

Reply:
just that `Arachi world` reads better than `Arach world` to me